books/templates/home.html

```diff
@@ -0,0 +1,127 @@
+{% extends "base.html" %}
```

Comment:
Remove this line and content block: can't extend from base.html because it was deleted